src/common/superagent.d.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { SuperAgentRequest } from 'superagent';
+declare module 'superagent' {
+  interface SuperAgentRequest {
+    // Guidance form Fusebit suggest that we should wait for requests to be
```

Comment:
nit: Typo `form` -> `from`. Probably negligible and not worth backporting to/from integrations packages

Reply:
Thanks.  Will fix here.  File probably won't change in integrations because the file will eventually be deleted there.

This should have been part of the original package.  Not sure how I missed it.

Reply:
Fixed.